scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -847,6 +871,30 @@ EOF
             # restart the execution of command if exit code is ESEGV error
             while [[ $cmd_exit_status == $ESEGV ]];
             do
+                #Following is the code to limit the number of core-m0beck and m0trace files, generated due to m0beck crash( receving SEGV signal ), to 2 each.
+                 run_cmd_on_remote_node "bash -s" <<EOF
+            core_m0beck_file_count=\$(cd /var/log/crash; find . -type f -exec stat -c "%n %y" {} \;| sort -n | grep core-m0beck | awk '{if(\$2>"'"${SCRIPT_START_DATE}"'") print \$2" "\$3; else if(\$2=="'"${SCRIPT_START_DATE}"'" && \$3>="'"${SCRIPT_START_TIME}"'") print \$2" "\$3; else echo }'| wc -l )
```

Comment:
Is Failover (Remote node not accessible) case handled?

Reply:
@RakeshVaghasiya Made the required changes for failover case with commit a01e9c5